pkg/monitor/alignchecker/alignchecker.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alignchecker
+
+import (
+	"reflect"
+
+	check "github.com/cilium/cilium/pkg/alignchecker"
+	"github.com/cilium/cilium/pkg/monitor"
+)
+
+// CheckStructAlignments checks whether size and offsets of the C and Go
```

Comment:
Nit: to avoid the comment getting out of sync, maybe you could say `See pkg/datapath/alignchecker/alignchecker.go:CheckStructAlignments() comment`?

Reply:
Good point, thanks. Done.